### ISSUE TITLE
Active Auctions not displaying

### ISSUE THREAD

**Issue by tamsinj53** [eq-p99-tools/ninjalooter#9]
While NLing plane of sky, if I had 3 active auctions; one of them (the last) would vanish.
- In pending: click dkp bid on an item
- when 2 active auctions, next dkp bid vanishes from sight
- when one of the 2 active auctions is completed, 3rd appears

This isn't completely consistent - three active bids have been visible.

**Comment by rm-you** [comment]
When it happens can you exit NL and post the state.json file or send it to me on discord? And after restarting the app see if it corrected itself or not. I'll see if I can reproduce the problem when I'm back at my computer early next week.

**Comment by rm-you** [comment]
Also, did this happen prior to the latest update (a few days ago) or is it newly observed behavior only?

**Comment by tamsinj53** [comment]
newly observed. I still have it open from having the issue - but the issue isn't present
actually, I'll unwind some bids now it's uploaded and see if i can reproduce

**Comment by tamsinj53** [comment]
0.7Mb state json.. that doesn't sound good to upload. will check what's in it

**Comment by rm-you** [comment]
> 0.7Mb state json.. that doesn't sound good to upload. will check what's in it

Yeah it is too big to paste but should be able to attach it? Probably nothing useful for you in there but having the entire thing will make it easier for me to cleanly reproduce your environment. Can easily send it in discord if not here.

**Comment by tamsinj53** [comment]
[problem.json](https://github.com/user-attachments/files/16737543/problem.json)
rom that state, undo belt of tranquility, undo sphinxian ring (okay)
undo spiritualist's ring. It does not move window.. just stays in completded - reclicking doesn't do anything. 
undo stein. also no visible effect

in fact, at that point nothing is working. no undo/completes have any visible effect 
/say Spiroc Wingblade - suddenly NL updates visually

**Comment by rm-you** [comment]
Are others having this issue as well? If this is very widespread and is causing an issue I will roll back the automatic update for this version and you can temporarily recommend people use the last release instead, though Senpai will not automatically be included for Castle raidticks. In your opinion is that warranted, or can you manage with this issue until Monday or Tuesday?

**Comment by tamsinj53** [comment]
I'm unaware of anyone else having the issue yet.
I can't make a rollback decision, sorry!

I suspect that without a quake it'll be totally fine.. maybe I'll ask the PM sky NL to be aware
